helm_deploy/monitoring/README.md

```diff
@@ -0,0 +1,93 @@
+
+
+#### EntityServiceImpl
+- [A class specifying CRU operations against metadata entities and aspects by primary key (urn)](https://github.com/datahub-project/datahub/blob/master/metadata-io/src/main/java/com/linkedin/metadata/entity/EntityServiceImpl.java#L107-L137)
```

Comment:
```suggestion
- [A class specifying CRUD operations against metadata entities and aspects by primary key (urn)](https://github.com/datahub-project/datahub/blob/master/metadata-io/src/main/java/com/linkedin/metadata/entity/EntityServiceImpl.java#L107-L137)
```

Reply:
Doesn't handle delete operations. Should I revert `CRU` to `Create, Read, and Update`?